src/newViews/Tagging/Export/ExportHistory/index.tsx

```diff
@@ -0,0 +1,248 @@
+    } = props;
+
+    const [activePage, setActivePage] = useState(1);
+    const [exportedAt, setExportedAt] = useState<DateRangeValue>();
+    const [searchText, setSearchText] = useState<string>();
+
+    const debouncedSearchText = useDebouncedValue(searchText);
+    const sortState = useSortState();
+    const { sorting } = sortState;
+    const validSorting = sorting || defaultSorting;
+    const ordering = useMemo(() => (
+        validSorting.direction === 'Ascending'
+            ? validSorting.name
+            : `-${validSorting.name}`
+    ), [validSorting]);
+
+    const {
+        pending,
+        response: exportListResponse,
+    } = useRequest<MultiResponse<Export>>({
+        url: 'server://exports/',
+        query: {
+            project: projectId,
+            ordering,
+            offset: (activePage - 1) * maxItemsPerPage,
+            limit: maxItemsPerPage,
+            title: debouncedSearchText, // FIXME: filter by search text is unsupported in server
+            exported_at__gte: exportedAt?.startDate && getDateWithTimezone(exportedAt.startDate),
+            exported_at__lt: exportedAt?.endDate && getDateWithTimezone(exportedAt.endDate),
+        },
+        method: 'GET',
+        schemaName: 'userExportsGetResponse',
+        shouldPoll: response => (
+            (response?.results.some(v => v.status === 'pending')) ? 5000 : -1
+        ),
+        failureHeader: 'Export History',
+    });
+
+    const handleDelete = useCallback((exportId: number) => { // TODO: add implementation later
+        console.warn('handleDelete', exportId);
+    }, []);
+
+    const columns = useMemo(() => {
+        const exportTypeColumn: TableColumn<
+        Export, number, IconsProps, TableHeaderCellProps
+        > = {
+            id: 'export_type',
+            title: 'Export Type',
+            headerCellRenderer: TableHeaderCell,
+            headerCellRendererParams: {
+                sortable: true,
+            },
+            cellRenderer: Icons,
+            cellRendererClassName: styles.icons,
+            cellRendererParams: (_, data) => ({
+                children: (
+                    <img
+                        className={styles.image}
+                        src={exportTypeIconMap[data.exportType]}
+                        alt={data.exportType}
+                    />),
+            }),
+        };
+        const statusColumn: TableColumn<
+        Export, number, TagProps, TableHeaderCellProps
+        > = {
+            id: 'status',
+            title: 'Status',
+            cellRendererClassName: styles.status,
+            headerCellRenderer: TableHeaderCell,
+            headerCellRendererParams: {
+                sortable: true,
+            },
+            cellRenderer: Tag,
+            cellRendererParams: (_, data) => ({
+                actions: statusIconMap[data.status],
+                variant: statusVariantMap[data.status],
+                children: statusLabelMap[data.status],
```

Comment:
We cannot download for the moment. Lets create a separate renderer for that to switch between options.